src/test/java/com/google/api/codegen/testdata/discoveries/php/php_sqladmin.v1beta4.json.baseline

```diff
@@ -1692,39 +1497,41 @@ echo '<pre>', var_export($response, true), '</pre>', "\n";
  *    https://console.developers.google.com/apis/api/sqladmin
  * 2. This sample uses Application Default Credentials for authentication.
  *    If not already done, install the gcloud CLI from
- *    https://cloud.google.com/sdk/ and run
+ *    https://cloud.google.com/sdk and run
  *    `gcloud beta auth application-default login`
+ *    For more information, see:
+ *    https://developers.google.com/identity/protocols/application-default-credentials
  * 3. Install the PHP client library with Composer. Check installation
  *    instructions at https://github.com/google/google-api-php-client.
  */
 
 // Autoload Composer.
 require_once __DIR__ . '/vendor/autoload.php';
 
-// Use Application Default Credentials for authentication when running locally.
-// For more information, see:
-// https://developers.google.com/identity/protocols/application-default-credentials
 $client = new Google_Client();
 $client->setApplicationName('Google-SQLAdminSample/0.1');
 $client->useApplicationDefaultCredentials();
 $client->addScope('https://www.googleapis.com/auth/cloud-platform');
 
 $service = new Google_Service_SQLAdmin($client);
 
-
-// TODO: Change placeholders below to desired parameter values for the `listUsers` method:
-
 // Project ID of the project that contains the instance.
-$project = '{MY-PROJECT}';
+$project = '';  // TODO: Update placeholder value.
 
 // Database instance ID. This does not include the project ID.
-$instance = '{MY-INSTANCE}';
+$instance = '';  // TODO: Update placeholder value.
 
+$optParams = [];
+do {
```

Comment:
Need override here: this method is not page-streaming. Do we have this in the other languages?